feature-libs/asm/components/customer-selection/customer-selection.component.ts

```diff
@@ -86,12 +96,15 @@ export class CustomerSelectionComponent implements OnInit, OnDestroy {
     }
   }
 
-  selectCustomerFromList(customer: User) {
+  selectCustomerFromList(event: UIEvent, customer: User) {
     this.selectedCustomer = customer;
     this.customerSelectionForm.controls.searchTerm.setValue(
       this.selectedCustomer.name
     );
     this.asmService.customerSearchReset();
+    this.searchTerm.nativeElement.focus();
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
These are usually invoked in the beginning of the function.
Also, I recall `event.stopImmediatePropagation` does both.

Reply:
i will test with stopImmediatePropagation

Reply:
stopImmediatePropagation doesn't work for Enter key.. it fires submit event